php/vagrant/provision.sh

```diff
@@ -63,7 +66,7 @@ apt-get install -y --no-install-recommends heroku-toolbelt ruby dos2unix man cur
 echo "ServerName ${SERVER}" >> /etc/apache2/apache2.conf;
 
 # Installing the PHP dev stack
-apt-get install -y --no-install-recommends git postgresql postgresql-contrib phpunit php5 php5-intl php5-pgsql php5-mcrypt php5-sqlite php5-apcu php5-cli
+apt-get install -y --no-install-recommends git postgresql postgresql-contrib phpunit php5 php5-intl php5-pgsql php5-mcrypt php5-sqlite php5-apcu php5-cli php5-gd
```

Comment:
You don't need `git`. `minimal/trusty64` comes pre-installed with `git`, `vim`, `opensshd`.